proto/dataplane/dataplane.proto

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package lemming.dataplane;
+
+option go_package = "github.com/openconfig/lemming/proto/dataplane";
+
+message UpdatePortRequest {
+  string name = 1;
+  string hwaddr = 2;
+  repeated IPNetwork ipv4 = 3;
+  repeated IPNetwork ipv6 = 4;
+}
+
+message UpdatePortResponse {}
+
+enum PortState {
+  PORTSTATE_INVALID = 0;
```

Comment:
nit: What's the purpose of INVALID here? I'm wondering because I'm thinking this might create extra handling code.

Reply:
the enum needs a default value and i wasn't convinced that DOWN is right as the default. I could rename this the PORTSTATE_UNSET if that's clearer

Reply:
I see, although I'm not sure what `PORTSTATE_UNSET` is the correct default either -- what does it even mean?

Reply:
Nvm, having an unset/unspecified value in an enum is in general good style, but if we ever add a new value that would cause trouble for backwards-compatibility.